template_feed/Microsoft.DotNet.Web.ProjectTemplates.2.0/content/RazorPagesWeb-CSharp/Pages/Account/Manage/TwoFactorAuthentication.cshtml

```diff
@@ -5,44 +5,32 @@
 }
 
 <h4>@ViewData["Title"]</h4>
-@if (Model.HasAny2faProviders)
+@if (Model.HasAny2faProviders && Model.Is2faEnabled)
```

Comment:
Should we just rename things to be clearer, 2fa => Authenticator?  And then this could just be a single Model.IsAuthenticatorEnabled check and we can do away with HasAny2faProviders entirely